test/test_mps.py

```diff
@@ -9446,6 +9447,75 @@ def test_fast_full_attention(self, dtype, contiguous, head_dim, with_mask):
         self.run_fast_attention_test(q, k, v, with_mask)
 
 
+
+
+class TestSDPAMetaDispatchMode(TorchDispatchMode):
```

Comment:
Is this test really necessary here? It feels like it's being indirectly tested by aot tests

Or, if we want an explicit test, shouldn't it be added to the `test_meta.py` or something?